### PR TITLE
✨ Use a single image for worker and controller

### PR DESCRIPTION
**Pull request by azeemshaikh38**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Only maintain a single image of worker/controller and use k8s args for running different jobs.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

### CODE REVIEW

File: cron/controller/Dockerfile [hunk 1]
```diff
@@ -27,5 +27,6 @@ RUN CGO_ENABLED=0 make build-pubsub
 
 FROM gcr.io/distroless/base:nonroot@sha256:bc84925113289d139a9ef2f309f0dd7ac46ea7b786f172ba9084ffdb4cbd9490
 COPY ./cron/data/projects.csv cron/data/projects.csv
+COPY ./cron/data/projects.release.csv cron/data/projects.release.csv
```
Comment (oliverchang) [inline cron/controller/Dockerfile:30]: nit: just `COPY ./cron/data/projects*csv cron/data/` to make it simpler and to avoid the extra layer.

**Review by oliverchang — APPROVED** [review]

File: cron/k8s/worker.release.yaml [hunk 2]
```diff
@@ -29,6 +29,7 @@ spec:
       containers:
       - name: worker
         image: gcr.io/openssf/scorecard-batch-worker-releasetest:latest
```
Comment (oliverchang) [inline cron/k8s/worker.release.yaml:31]: can we share the worker image for release test as well?

File: cron/controller/Dockerfile (hunk 1 above)
Comment (azeemshaikh38) [reply cron/controller/Dockerfile]: Done.

File: cron/k8s/worker.release.yaml (hunk 2 above)
Comment (azeemshaikh38) [reply cron/k8s/worker.release.yaml]: Ah forgot. Done.

**Pull request merged**